src/oss-metadata/MetadataTool.cs

```diff
@@ -54,7 +54,7 @@ public MetadataTool(IHttpClientFactory httpClientFactory) : base(httpClientFacto
                 BaseProjectManager? projectManager = ProjectManagerFactory.CreateProjectManager(purl, httpClientFactory);
                 if (projectManager != null)
                 {
-                    metadata = await projectManager.GetPackageMetadata(purl, false);
+                    metadata = await projectManager.GetPackageMetadata(purl, useCache: false);
```

Comment:
This shouldn't be hard coded false. Might need to introduce a parameter.

Reply:
Added it to the options.